### PR TITLE
Add favicon

### PR DESCRIPTION
**Pull request by KaratasFurkan**
Hi, I recently saw [this issue](https://github.com/classy-python/ccbv/issues/140) and I was also wishing to have a favicon, mostly to distinguish [ccbv.co.uk](https://ccbv.co.uk/) and localhost :smile:

So I created and added a favicon, nothing fancy but I hope you like it :slightly_smiling_face:

### CODE REVIEW

**Comment by meshy** [comment]
Yes, favicons are tricky! Thankfully I think using an SVG should cut out most of the hassle on [most major browsers with the exception of Safari](https://caniuse.com/?search=svg%20favicon), which I'm happy to use a single PNG for.

I could be wrong, but I believe the manifest file might only be required on Android when setting up a link to the website as an icon on the homescreen, and I suspect that's fairly rare for this site. I think we can skip it for now, unless you find otherwise.

**Comment by KaratasFurkan** [comment]
Thanks for the comments @meshy, I agree about the android part. I'm creating the svg icon, I'll update the PR accordingly :+1:

File: cbv/templates/base.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
     <meta name="author" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <!-- Favicon -->
+    <link rel="icon" href="{% static 'favicon/favicon.ico' %}" sizes="48x48" />
```
Comment (KaratasFurkan) [inline cbv/templates/base.html:12]: `favicon.ico` is added as a backup for Safari. Recommended in `2. The .ico favicon (for Safari and IE)` section of https://medium.com/web-dev-survey-from-kyoto/favicon-nightmare-how-to-maintain-sanity-7628bfc39918

Other browsers pick up svg icon.

**Comment by KaratasFurkan** [comment]
Hey @meshy, the SVG icon is ready. I also updated the font to Iosevka, since it's characters are taller, the text is more visible. Also, it is my favorite font for coding :smile:

Here is the new version for a quick check:

favicon

> I could be wrong, but I believe the manifest file might only be required on Android when setting up a link to the website as an icon on the homescreen, and I suspect that's fairly rare for this site. I think we can skip it for now, unless you find otherwise.

Android & iOS homescreen icons are removed as you suggested :+1:

**Comment by KaratasFurkan** [comment]
As a reference, this is the simpler version of the SVG code I used to generate this image:
```xml
<svg height="512" width="512" xmlns="http://www.w3.org/2000/svg">
  <rect width="512" height="512" rx="20" ry="20" fill="#092E20" />
  <text font-family="Iosevka" font-weight="bold" font-size="300" textLength="515" x="1" y="366">
    <tspan fill="#CFB955">C</tspan><tspan fill="#FFF">CBV</tspan>
  </text>
</svg>
```

Since it's not supported on many places this way, I converted the `rect` & `text` elements to `path`s with Inkscape. So that's how I created the final version.

**Comment by meshy** [comment]
Thank you!

**Pull request merged**